vuu-ui/packages/vuu-data/src/array-data-source/array-data-source.ts

```diff
@@ -40,6 +44,8 @@ import {
 } from "../vuuUIMessageTypes";
 import { collapseGroup, expandGroup, GroupMap, groupRows } from "./group-utils";
 import { sortRows } from "./sort-utils";
+import { el } from "@faker-js/faker";
```

Comment:
don't think we are using faker in here, are we ?

Reply:
sorry about it! Forgot to change the tag to WIP -- shifting faker to another file to make it more modular